### ISSUE TITLE
How to Start Freelancing

### ISSUE THREAD

**Issue by JMo911** [coryhouse/ama#35]
Hello Cory,

I've been working as a software engineer for large corporations for 4 years now and am looking to transition into freelance/contracting. I have several questions on how to get started. Would greatly appreciate any guidance you can offer with the following:

1. What was your journey to freelance/consulting?
2. How did you get your first independent gig?
3. How did you find work/clients when you were getting started? 
4. What do you use for benefits? Do you have international benefits?
5. How do you calculate your rate?
6. What lessons have you learned along the way? 
    6a. Things you wish you had done differently?
    6b. Things you wish you knew when you were getting started?
    6c. Things to avoid in freelancing?
7. What did your portfolio (bitnative) look like when you were first getting started? Why?

**Comment by coryhouse** [comment]
Hi Jacob,

1. I kept my job as long as I could. I built a business on the side by authoring, training, and consulting nights and weekends. I only quit when I had a full funnel, and confidence I could keep it full.
2. Through a contracting company that contacted me.
3. I spoke at conferences, blogged, tweeted, authored courses, and did podcasts. I marketed myself as a React specialist in these venues.
4. I contacted a local specialist for help on picking a healthcare plan.
5. I talked money with fellow contractors at conferences and user groups. Today, I ladder my rate. When I'm too busy, I charge more. When I have too much slack (thankfully, this has been very rare), I charge less.
6. a - I wish I'd specialized earlier, and I should have gotten an accountant sooner. b - How to handle invoices, taxes, statements of work, contracts. c - I avoid doing more than 20 hours with a client. This avoids putting all my eggs in one basket.
7. It was a mixture of training, consulting, development, and authoring courses with 2-6 clients/week. This continues today.

Hope that helps!
